pkg/config/schema/codegen/collections.go

```diff
@@ -67,8 +68,11 @@ var (
 			Plural: "{{ .Resource.Plural }}",
 			Version: "{{ .Resource.Version }}",
 			Proto: "{{ .Resource.Proto }}",
+			{{ if ne .Resource.StatusProto "" }}StatusProto: "{{ .Resource.StatusProto }}",{{end}}
```

Comment:
```suggestion
			{{- if ne .Resource.StatusProto "" }}StatusProto: "{{ .Resource.StatusProto }}",{{end}}
```
Not a big deal, but it means it will not insert a blank line if it is false